subprojects/jvm-services/src/main/java/org/gradle/jvm/toolchain/internal/MavenToolchainsInstallationSupplier.java

```diff
@@ -85,7 +88,18 @@ protected Set<InstallationLocation> findCandidates() {
                 for (int i = 0; i < nodes.getLength(); i++) {
                     Node item = nodes.item(i);
                     if (item != null && item.getNodeType() == Node.TEXT_NODE) {
-                        locations.add(item.getNodeValue().trim());
+                        String location = item.getNodeValue().trim();
+                        Matcher envMatches = ENV_VAR_PATTERN.matcher(location);
+                        if(envMatches.matches()) {
```

Comment:
According to the surrounding code style:
```suggestion
                        if (envMatches.matches()) {
```

Reply:
@valery1707 thanks for pointing this out. I fixed it. Because it was a simple whitespace error I amended the previous self-contained commit to do so instead of creating a new one. I hope this works for you – the [contributing guide](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md) doesn't say what is preferred as far as I can tell.

Reply:
Yes, amending the original commit is preferred in my opinion.

In any case, I was just passing by and suggesting some minor improvements to the codebase and have no way of approving or promoting this PR.